ui/raidboss/data/06-ew/raid/p8s.ts

```diff
@@ -1732,6 +1737,57 @@ const triggerSet: TriggerSet<Data> = {
       netRegex: NetRegexes.startsUsing({ id: '75ED', source: 'Hephaistos', capture: false }),
       response: Responses.spread('alert'),
     },
+    {
+      id: 'P8S Tyrant\'s Fire III Counter',
+      type: 'Ability',
+      netRegex: NetRegexes.ability({ id: '75F0', source: 'Hephaistos', capture: false }),
+      preRun: (data) => data.burstCounter++,
+      durationSeconds: 3,
+      suppressSeconds: 1,
+      sound: '',
+      infoText: (data, _matches, output) => output.text!({ num: data.burstCounter }),
+      tts: null,
+      outputStrings: {
+        text: {
+          en: '${num}',
+          de: '${num}',
+          fr: '${num}',
+          ja: '${num}',
+          cn: '${num}',
+          ko: '${num}',
+        },
+      },
+    },
+    {
+      id: 'P8S Tyrant\'s Fire III Bait then Tower',
+      type: 'Ability',
+      netRegex: NetRegexes.ability({ id: '75F0', source: 'Hephaistos' }),
+      condition: Conditions.targetIsYou(),
+      durationSeconds: 7.9,
+      infoText: (data, _matches, output) => output.text!({ num: data.burstCounter }),
+      run: (data) => data.myTower = data.burstCounter,
+      outputStrings: {
+        text: {
+          en: '${num}: Bait near Tower ${num}',
```

Comment:
```suggestion
      alertText: (data, _matches, output) => output.text!({ num: data.burstCounter }),
      run: (data) => data.myTower = data.burstCounter,
      outputStrings: {
        text: {
          en: '${num}',
```

I think this should be an alert to not overlap with all the infoText going on.  I also think this is a "you need to understand the mechanic".  Just the number is enough.

Reply:
Works for me.